spec/controllers/catalog_controller_spec.rb

```diff
@@ -23,6 +23,7 @@ def assigns_response
     end
   
     it "should have search results for a poor query with multiple key words" do
+      sign_in User.create({:provider => 'developer'})
```

Comment:
@mcritchlow  I think the test object is created in curator view and this test is examining the the search result, while the other test won't.